Makefile

```diff
@@ -449,6 +451,123 @@ cawiki_tuning_reports: \
 	tuning_reports/cawiki.damaging.md \
 	tuning_reports/cawiki.goodfaith.md
 
+############################# Czech Wikipedia ################################
+
+datasets/cswiki.sampled_revisions.20k_2016.json:
+	wget -qO- https://quarry.wmflabs.org/run/97125/output/0/json-lines?download=true > $@
+
+datasets/cswiki.autolabeled_revisions.20k_2016.json: \
+		datasets/cswiki.sampled_revisions.20k_2016.json
+	cat $< | \
+	./utility autolabel --host=https://cs.wikipedia.org \
+		--trusted-groups=sysop,oversight,editor,bot,rollbacker,checkuser,abusefilter,bureaucrat \
+		--trusted-edits=1000 \
+		--revert-radius=3 \
+		--revert-window=48 \
+		--verbose > $@
+
+datasets/cswiki.human_labeled_revisions.5k_2016.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/cswiki/44/ > $@
+
+datasets/cswiki.revisions_for_review.5k_2016.json: \
+		datasets/cswiki.autolabeled_revisions.20k_2016.json
+	( \
+	 cat $< | \
+	 grep '"needs_review": true' | \
+	 shuf -n 2500; \
+	 cat $< | \
+	 grep '"needs_review": false' | \
+	 shuf -n 2500 \
+	) | shuf > $@
+
+datasets/cswiki.labeled_revisions.20k_2016.json: \
+		datasets/cswiki.human_labeled_revisions.5k_2016.json \
+		datasets/cswiki.autolabeled_revisions.20k_2016.json
+	./utility merge_labels $^ > $@
```

Comment:
This is a problem because we have "needs_review" with no corresponding labeled data in `datasets/cswiki.autolabeled_revisions.20k_2016.json` due to our sub-sampling.

Reply:
True, but this is already happening, for example in case of French Wikipedia. I'm not sure if I broke it or it was broken before too. What's the best approach here? :(

Reply:
In the old version, there was a separate merge for needs_review and not needs_review.  That's how we addressed this issue.

Reply:
Essentially, here's what I have in mind:  

1. merge_labels autolabeled (!needs_review) with human_labeled (!needs_review)
2. cat the above with human_labeled (needs_review)
3. Write that out as the labeled data.  

Essentially, we're ignoring the autolabeled (needs_review) that was not human_labeled.